srcCxx/ShapeType_main.cxx

```diff
@@ -368,16 +370,17 @@ int run(DomainId_t domain_id, bool use_security,
 void print_usage( const char * name )
 {
   printf( "Usage:  %s\n", name );
-  printf( "    [-pub <pubTopic>]                :  default: '%s'\n", DEFAULT_PUBLISH_TOPIC );
-  printf( "    [-sub <subTopic>]                :  default: '%s'\n", DEFAULT_SUBSCRIBE_TOPIC );
-  printf( "    [-domain <domainId>]             :  default: '%d'\n", DEFAULT_DOMAIN_ID );
-  printf( "    [-color <colorName>]             :  default: '%s'\n", DEFAULT_COLOR );
-  printf( "    [-governance <governanceFile>]   :  default: '%s'\n", DEFAULT_GOVERNANCE_FILE );
-  printf( "    [-permissions <permissionsFile>] :  default: '%s'\n", DEFAULT_PERMISSIONS_FILE );
-  printf( "    [-partition <partitionStr>]      :  default: '%s'\n", "<empty>" );
-  printf( "    [-livelinessPeriod <float>]      :  default: %f\n",   DEFAULT_LIVELINESS_PERIOD );
-  printf( "    [-disableSecurity]               :  default: %s\n",   "false" );
-  printf( "    [-logging]                       :  default: %s\n",   "false" );
+  printf( "    [-pub <pubTopic>]                 :  default: '%s'\n", DEFAULT_PUBLISH_TOPIC );
+  printf( "    [-sub <subTopic>]                 :  default: '%s'\n", DEFAULT_SUBSCRIBE_TOPIC );
+  printf( "    [-domain <domainId>]              :  default: '%d'\n", DEFAULT_DOMAIN_ID );
+  printf( "    [-color <colorName>]              :  default: '%s'\n", DEFAULT_COLOR );
+  printf( "    [-governance <governanceFile>]    :  default: '%s'\n", DEFAULT_GOVERNANCE_FILE );
+  printf( "    [-permissions <permissionsFile>]  :  default: '%s'\n", DEFAULT_PERMISSIONS_FILE );
+  printf( "    [-kest <keyEstablishmentAlgo>]    :  default: '%s'\n", DEFAULT_KEY_EST_ALGORITHM);
```

Comment:
```suggestion
  printf( "    [-kest <authentication.key_establishment_algorithm>]    :  default: '%s'\n", DEFAULT_KEY_EST_ALGORITHM);
```
so that people can search for this property in [our documentation](https://community.rti.com/static/documentation/connext-dds/7.3.0/doc/manuals/connext_dds_secure/users_manual/p2_core/authentication.html#rti-sp-product-heading-properties-for-configuring-authentication).

Reply:
@YushengYangRTI This code is common to all vendors, so I don't think we should point to a rti-specific property here.

Reply:
We could use "dds.sec.auth.key_establishment_algorithm" instead of "authentication. key_establishment_algorithm".
The v1.2 spec has formalized that name.